### ISSUE TITLE
Pedidos pagos via PIX não são marcados pagos

### ISSUE THREAD

**Issue by jffaria** [claudiosanches/woocommerce-pagseguro#133]
Oi pessoal,

Recentemente o PagSeguro introduziu a opção de pagar via PIX, e notei que quando os clientes utilizam esta opção é adicionado um comentário no pedido indicando que o pedido foi pago, porém o status não é atualizado.

Abraço

**Comment by gabsnery** [comment]
@jffaria Estou com o mesmo problema
Conseguiu resolver de alguma forma?

**Comment by jffaria** [comment]
@gabsnery sim, eu fiz uma alteração no código do plugin para quebrar o galho pra mim aqui. Esses dias postei lá no fórum do plugin no wordpress.org: https://wordpress.org/support/topic/pagamento-pix/#post-13828465 é só copiar e colar e deve resolver. Fiquei de mandar um PR pro Cláudio mas ainda não tive tempo de fazer algo mais redondo.

**Comment by fredroo** [comment]
@jffaria poderia mandar a PR aqui, também queria resolver o problema via atualização do plugin e não com código externo

desde já agradeço

**Comment by fredroo** [comment]
@claudiosanches olha essa solução do pix veja se ela atende o plugin

**Comment by fredroo** [comment]
@jffaria tem como mandar a PR para facilitar

**Comment by clebersleite** [comment]
Alguém me informa, onde tem essa opção de PIX nesse plugin???

**Comment by fredroo** [comment]
https://github.com/r-martins/woocommerce-rmpagseguro/pull/1

@claudiosanches

**Comment by fredroo** [comment]
@clebersleite so precisei atualizar o plugin e ela já ficou disponível, mas tem que dar baixa manualmente, por isso essa issue
